content/token22-in-the-client.md

```diff
@@ -0,0 +1,470 @@
+---
+title: Supporting Token Extension Program from a Client
```

Comment:
Let's change these titles to be declarative rather than use the gerund, e.g. "Support the Token Extensions Program from a Client" instead of "Supporting..."

Reply:
Imperative?? Idk...words